src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}

```

Comment:
The class `Machine` should be public to ensure it can be accessed from other packages, which is likely needed for an abstract class meant to be subclassed.